graze/ciff-renderer
# Fallback to default font size if none is set - OPS-1938

### Summary
If the font size of a text field is set to 13, a `pitch` value is not set. I assume this is the format's default font size. This PR replicates that behaviour.

### Notable Changes
- Default to pitch 13 if no pitch is present
- Handle ciffs that contain only a single field

### QA
- Replace the contents of `/demo/demo.ciff` with that of this file [demo.ciff.txt](https://github.com/graze/ciff-renderer/files/1915321/demo.ciff.txt)
- `php -S localhost:8000`, start PHP's dev webserver
- Browse to http://localhost:8000/demo/demo.php
- Ensure you are presented with the text `This is some text of size 13`
